frontend/src/components/design-system/navigation/NavigationBar.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import ContainerComponent from '@/components/design-system/containers/ContainerComponent.vue';
+import StackComponent from '@/components/design-system/layout/StackComponent.vue';
+import LogoComponent from '@/components/design-system/style/LogoComponent.vue';
+import BodyText from '@/components/design-system/typography/BodyText.vue';
+
+export interface Props {
+  transparent?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  transparent: false,
+});
+
+const background = computed(() =>
+  props.transparent ? 'transparent' : undefined
+);
+</script>
+
+<template>
+  <ContainerComponent
+    class="navigation-bar"
+    :background="background"
+    padding="medium"
+  >
+    <div class="grid">
+      <LogoComponent />
+      <div class="title">
+        <BodyText size="small">
+          <slot name="title" />
+        </BodyText>
+      </div>
+      <StackComponent class="right" horizontal>
+        <slot name="right" />
+      </StackComponent>
+    </div>
+  </ContainerComponent>
```

Comment:
Chybí mi tu ještě jeden řádek. Např. rekapitulace, výsledky a porovnání mají řádky dva..

Reply:
To bude jiná komponenta.